test_runner/flank-invalid.yml

```diff
@@ -0,0 +1,6 @@
+gcloud:
+  app: ../test_app/apks/app-debug.apk
+  test: ../test_app/apks/app-debug-androidTest.apk
+
+flank:
+  incorrect: fields
```

Comment:
![image](https://user-images.githubusercontent.com/1173057/75410787-dbf69180-58d1-11ea-9b73-48d3e2cece85.png)
where possible, end the files with a newline :)